modules/tinymce/src/core/test/ts/browser/UndoManagerTest.ts

```diff
@@ -21,7 +21,7 @@ describe('browser.tinymce.core.UndoManagerTest', () => {
     indent: false,
     entities: 'raw',
     base_url: '/project/tinymce/js/tinymce'
-  }, []);
+  }, [], true);
```

Comment:
This is worrying... why was this added and did the test also fail on `develop` without this? If it didn't then we likely have a bug in these changes.